lib/filedemuxer.js

```diff
@@ -59,6 +59,11 @@ inherits(FileDemuxer, EventEmitter);
  * @private
  */
 FileDemuxer.prototype._openStream = function() {
+  if (this._fileSize === 0) {
+    this._currentOutput = new stream.Readable({ read: utils.noop });
+    this.emit('error', new Error('File size cannot be 0 Bytes.'));
```

Comment:
I'd add a `return` in front of `this.emit('error', ...)` so we don't fall through and set up event listeners on the dead stream.